cmd/operator-sdk/add/api.go

```diff
@@ -33,24 +33,27 @@ import (
 var (
 	apiVersion string
 	kind       string
+	k8sCodegen bool
+	openAPIGen bool
 )
 
 func newAddApiCmd() *cobra.Command {
 	apiCmd := &cobra.Command{
 		Use:   "api",
 		Short: "Adds a new api definition under pkg/apis",
-		Long: `operator-sdk add api --kind=<kind> --api-version=<group/version> creates the
+		Long: `operator-sdk add api --kind=<kind> --api-version=<group/version> --k8s-codegen=<true or false> --openapi-gen=<true or false> creates the
```

Comment:
HI @yangzhares,

Really tks for the collab 🥇.  I would keep the current nomenclature which is: 

```
	operator-sdk generate k8s
	operator-sdk generate openapi
```
So, my suggestion is to replace --k8s-codegen for k8s and  `--openapi-gen` for `openapi`. Also, if be default it is true then in the explanation would make sense in IMHO have just the boolean value false.